### ISSUE TITLE
App crash if Modal Page is opened

### ISSUE THREAD

**Issue by webleaf** [NativeScript/nativescript-cli#3755]
If some XML or CSS file is change and in app Modal Page is opened (was opened on touch event), app crash during LiveSync process.

```
System.err: com.tns.NativeScriptException: 
System.err: Calling js method onDismiss failed
System.err: 
System.err: TypeError: Cannot read property 'setOnTouchListener' of null
System.err: File: "file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view.js, line: 213, column: 37
System.err: 
System.err: StackTrace: 
System.err: 	Frame: function:'View.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 213, column: 38
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err: 	Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err: 	Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err: 	Frame: function:'ViewBase.unloadView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 246, column: 19
System.err: 	Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err: 	Frame: function:'ViewCommon.eachChild', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err: 	Frame: function:'ViewBase.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 245, column: 14
System.err: 	Frame: function:'View.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 218, column: 37
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err: 	Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err: 	Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err: 	Frame: function:'ViewBase.unloadView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 246, column: 19
System.err: 	Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err: 	Frame: function:'ViewCommon.eachChild', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err: 	Frame: function:'ViewBase.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 245, column: 14
System.err: 	Frame: function:'View.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 218, column: 37
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err: 	Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err: 	Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err: 	Frame: function:'ViewBase.unloadView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 246, column: 19
System.err: 	Frame: function:'ContentView.eachChildView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/content-view/content-view.js', line: 70, column: 13
System.err: 	Frame: function:'PageBase.eachChildView', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/page/page-common.js', line: 120, column: 40
System.err: 	Frame: function:'ViewCommon.eachChild', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err: 	Frame: function:'ViewBase.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 245, column: 14
System.err: 	Frame: function:'View.onUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 218, column: 37
System.err: 	Frame: function:'', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err: 	Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err: 	Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err: 	Frame: function:'DialogFragmentImpl.onDismiss', file:'file:///data/data/com.test/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 137, column: 23
System.err: 
System.err: 	at com.tns.Runtime.callJSMethodNative(Native Method)
System.err: 	at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1101)
System.err: 	at com.tns.Runtime.callJSMethodImpl(Runtime.java:983)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:970)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:954)
System.err: 	at com.tns.Runtime.callJSMethod(Runtime.java:946)
System.err: 	at com.tns.gen.android.app.DialogFragment_view_84_32_DialogFragmentImpl.onDismiss(DialogFragment_view_84_32_DialogFragmentImpl.java:36)
System.err: 	at android.app.Dialog$ListenersHandler.handleMessage(Dialog.java:1477)
System.err: 	at android.os.Handler.dispatchMessage(Handler.java:102)
System.err: 	at android.os.Looper.loop(Looper.java:154)
System.err: 	at android.app.ActivityThread.main(ActivityThread.java:6776)
System.err: 	at java.lang.reflect.Method.invoke(Native Method)
System.err: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1518)
System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1408)
ActivityManager: Process com.test (pid 20808) has died(61,541)
ActivityManager: cleanUpApplicationRecord -- 20808
```

```
$tns --version
4.1.2
```
from package.json:
```
"nativescript": {
    "id": "com.test",
    "tns-android": {
      "version": "4.1.3"
    }
  }
```

**Comment by NickIliev** [comment]
@webleafI've tried to reproduce the issue but to no avail. Here is [the test project](https://github.com/NickIliev/NS-Issues-2018-II/tree/master/nativescript-cli/issue-3755) I am using - please take a look at it and let me know what is the specific case/code base needed to reproduce the issue.

**Comment by vahidvdn** [comment]
@NickIliev This also happen to me. I don't know when the error occurs, but it happens sometimes. Error log:
```
System.err: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.nativescript.tns423/com.tns.NativeScriptActivity}: com.tns.NativeScriptException:
System.err: Calling js method onCreate failed
System.err:
System.err: TypeError: Cannot read property 'setOnTouchListener' of null
System.err: File: "file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view.js, line: 213, column: 37
System.err:
System.err: StackTrace:
System.err:     Frame: function:'View.onUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 213, column: 38
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err:     Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err:     Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err:     Frame: function:'ViewBase.unloadView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 246, column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase.onUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 245, column: 14
System.err:     Frame: function:'View.onUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 218, column: 37
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err:     Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err:     Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err:     Frame: function:'ViewBase.unloadView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err:     Frame: function:'ViewBase._removeViewCore', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 462, column: 14
System.err:     Frame: function:'ViewBase._removeView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 457, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/list-view/list-view.js', line: 144, column: 27
System.err:     Frame: function:'ListView.clearRealizedCells', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/list-view/list-view.js', line: 141, column: 29
System.err:     Frame: function:'ListView.disposeNativeView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/list-view/list-view.js', line: 79, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 564, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'ContentView.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/content-view/content-view.js', line: 70, column: 13
System.err:     Frame: function:'PageBase.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/page/page-common.js', line: 120, column: 40
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'FrameBase.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/frame-common.js', line: 340, column: 13
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 486, column: 18
System.err:     Frame: function:'ViewBase._setupAsRootView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 479, column: 14
System.err:     Frame: function:'ViewCommon._setupAsRootView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 98, column: 43
System.err:     Frame: function:'ActivityCallbacksImplementation.setActivityContent', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 836, column: 22
System.err:     Frame: function:'ActivityCallbacksImplementation.onCreate', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 679, column: 14
System.err:     Frame: function:'NativeScriptActivity.onCreate', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/activity.js', line: 20, column: 25
System.err:
System.err:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2793)
System.err:     at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2864)
System.err:     at android.app.ActivityThread.-wrap12(ActivityThread.java)
System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1567)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:105)
System.err:     at android.os.Looper.loop(Looper.java:156)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:6523)
System.err:     at java.lang.reflect.Method.invoke(Native Method)
System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:941)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:831)
System.err: Caused by: com.tns.NativeScriptException:
System.err: Calling js method onCreate failed
System.err:
System.err: TypeError: Cannot read property 'setOnTouchListener' of null
System.err: File: "file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view.js, line: 213, column: 37
System.err:
System.err: StackTrace:
System.err:     Frame: function:'View.onUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 213, column: 38
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err:     Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err:     Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err:     Frame: function:'ViewBase.unloadView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 246, column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase.onUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 245, column: 14
System.err:     Frame: function:'View.onUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view.js', line: 218, column: 37
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 92
System.err:     Frame: function:'ViewBase.callFunctionWithSuper', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 304, column: 9
System.err:     Frame: function:'ViewBase.callUnloaded', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 315, column: 14
System.err:     Frame: function:'ViewBase.unloadView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 444, column: 18
System.err:     Frame: function:'ViewBase._removeViewCore', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 462, column: 14
System.err:     Frame: function:'ViewBase._removeView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 457, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/list-view/list-view.js', line: 144, column: 27
System.err:     Frame: function:'ListView.clearRealizedCells', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/list-view/list-view.js', line: 141, column: 29
System.err:     Frame: function:'ListView.disposeNativeView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/list-view/list-view.js', line: 79, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 564, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'LayoutBaseCommon.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/layouts/layout-base-common.js', line: 125, column: 26
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'ContentView.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/content-view/content-view.js', line: 70, column: 13
System.err:     Frame: function:'PageBase.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/page/page-common.js', line: 120, column: 40
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 558, column: 19
System.err:     Frame: function:'FrameBase.eachChildView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/frame-common.js', line: 340, column: 13
System.err:     Frame: function:'ViewCommon.eachChild', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 868, column: 14
System.err:     Frame: function:'ViewBase._tearDownUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 557, column: 14
System.err:     Frame: function:'ViewBase._setupUI', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 486, column: 18
System.err:     Frame: function:'ViewBase._setupAsRootView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view-base/view-base.js', line: 479, column: 14
System.err:     Frame: function:'ViewCommon._setupAsRootView', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/core/view/view-common.js', line: 98, column: 43
System.err:     Frame: function:'ActivityCallbacksImplementation.setActivityContent', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 836, column: 22
System.err:     Frame: function:'ActivityCallbacksImplementation.onCreate', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/frame.js', line: 679, column: 14
System.err:     Frame: function:'NativeScriptActivity.onCreate', file:'file:///data/data/org.nativescript.tns423/files/app/tns_modules/tns-core-modules/ui/frame/activity.js', line: 20, column: 25
System.err:
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1116)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:996)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:983)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:967)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:959)
System.err:     at com.tns.NativeScriptActivity.onCreate(NativeScriptActivity.java:18)
System.err:     at android.app.Activity.performCreate(Activity.java:6910)
System.err:     at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1123)
System.err:     at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2746)
System.err:     ... 9 more
```

**Comment by vahidvdn** [comment]
@NickIliev I simply create a blank ng project and run to my device. Btw, I share my project and info:
```
√ Getting NativeScript components versions information...
√ Component nativescript has 4.2.2 version and is up to date.
√ Component tns-core-modules has 4.2.0 version and is up to date.
√ Component tns-android has 4.2.0 version and is up to date.
√ Component tns-ios has 4.2.0 version and is up to date.
```
[tns42.zip](https://github.com/NativeScript/nativescript-cli/files/2314785/tns42.zip)

**Comment by NickIliev** [comment]
@vahidvdn the project is working as expected on my side - try clean rebuild and also can you please share with us what is device you are using for tests.

**Comment by vahidvdn** [comment]
@NickIliev At the beginning it works fine. But after some changes, I get the errors.

**Comment by nuvoPoint** [comment]
I have the same issue on Samsung S8

**Comment by webleaf** [comment]
Similar issue occurs if you try open modal page B inside opened modal page A, but immediately after tapping button "open B" (and before modal page B opened in fact) tap button "close A".

**Comment by mayureshjadhav** [comment]
After digging I found that the issue was in "nativescript-dev-webpack" plugin. If anyone is facing the issue, please update the plugin.

**Comment by surdu** [comment]
We managed to find a scenario on how to reproduce it:

- open a modal
- put the app in background
- get a notification from Firebase
- tap the notification, which will open the app
- the app will crash with the above error

The crash only happens when the app is reopened with the notification, not if you tap the app icon or switch to the app. This make me believe this is related to [this other issue](https://github.com/NativeScript/NativeScript/issues/4981#issuecomment-347526864), because that issue will reproduce when you [tap the "Open" button](https://github.com/NativeScript/NativeScript/issues/4981#issuecomment-347533604) but also when you open the app via a Firebase notification.

I've tried with a app-created notifications, and it seems it's not the same. I'm guessing the Firebase notification are created by the OS and in a different way than the notifications the app creates.
